src/marlin_stubs/host/M115.cpp

```diff
@@ -43,14 +43,14 @@ void GcodeSuite::M115() {
     SERIAL_ECHOPGM("FIRMWARE_NAME:Prusa-Firmware-Buddy ");
     SERIAL_ECHOPGM(project_version_full);
     SERIAL_ECHOPGM(" (Github) SOURCE_CODE_URL:https://github.com/prusa3d/Prusa-Firmware-Buddy");
-#if PRINTER_IS_PRUSA_MK4
-    SERIAL_ECHOLNPGM(" PROTOCOL_VERSION:" PROTOCOL_VERSION " MACHINE_TYPE:"
-                     "MK4"
-                     " EXTRUDER_COUNT:" STRINGIFY(1) " UUID:" MACHINE_UUID); // extruder hardcoded due to mmu, should be dynamic in the future
+    SERIAL_ECHOLNPGM(" PROTOCOL_VERSION:" PROTOCOL_VERSION " MACHINE_TYPE:" PRINTER_MODEL
+                     " EXTRUDER_COUNT:"
+#ifdef HAS_MMU2()
+        STRINGIFY(1)
```

Comment:
This would deserve an explaining comment

Reply:
I've added comments.

Reply:
The comment is not descriptive enough, sorry. It should also explain why EXTRUDERS has a different value.

Reply:
Ok, I've added a more descriptive comment, however that is my opinion as I understand the code and might not reflect how it should be understood.